### ISSUE TITLE
[Feature request] Note aliases

### ISSUE THREAD

**Issue by wwwald** [zadam/trilium#1604]
I have a "contacts" subtree in my wiki, with notes for a number of work contacts. Note title is the person's name, for example,  there could be "zadam" and "wwwald" notes in there.

Now, some contacts have names with special characters, a bit harder to type on my keyboard layout. For example, a name like "Jønȃs MádeÜpLãstNæme" (made up, obviously).

Now, I would like to be able to just link to this page by typing "`@jonas...`" without all the special characters, and selecting the contact note. If I could define a note alias "Jonas MadeUpLastName", I could use that to refer to this page. The final note title shown everywhere will still be the original one, with the accents.

So basically, I would have a note titled "Jønȃs MádeÜpLãstNæme" with  an `#alias="Jonas MadeUpLastName"` attribute, allowing me to link to any of those title variations.
This would not only be useful for hard-to-type contact names, but I could imagine some other uses too.

The idea is stolen from Wikidpad, that allows to define `[alias: ...]` tags in wiki notes.

What do you think?

**Comment by zadam** [comment]
Hi, if I understand you correctly it's mainly about being able to search by the alias, in that case this already works since search / autocomplete look for matches into the attributes:

![](https://user-images.githubusercontent.com/617641/106803413-7aec4780-6664-11eb-8f68-9bc39ecb6f5b.png)

**Comment by wwwald** [comment]
OK! Good to know, that indeed works to fix my name problem!

**Comment by sc8cn** [comment]
Adding an alias attribute to the software and referencing the alias directly when @ referencing is a very useful feature that I have used in obsidian. Some excerpts and other types of article titles especially require aliases
